PWGCF/FemtoDream/TableProducer/femtoDreamProducerTask.cxx

```diff
@@ -92,6 +101,13 @@ struct femtoDreamProducerTask {
   Configurable<bool> ConfEvtAddOfflineCheck{"ConfEvtAddOfflineCheck", false, "Evt sel: additional checks for offline selection (not part of sel8 yet)"};
   Configurable<bool> ConfIsActivateV0{"ConfIsActivateV0", true, "Activate filling of V0 into femtodream tables"};
 
+  /// Lc table
+  Configurable<bool> useCent{"useCent", false, "Enable centrality for lc"};
+  Configurable<bool> fillLcMCGenTable{"fillLcMCGenTable", false, "Enable mc gen table for lc"};
+  Configurable<bool> keepOnlySignalMc{"keepOnlySignalMc", false, "Fill MC tree only with signal candidates"};
+  Configurable<bool> keepOnlyBkg{"keepOnlyBkg", false, "Fill MC tree only with background candidates"};
```

Comment:
I am confident that we should not keep 'keepOnlyBkg', but I'm uncertain whether we should retain both signal and background or just the signal. While I understand that we typically use only the signal for model training, I'm unsure if the same applies for femtoscopy. What are your thoughts on this?

Reply:
In general, since this is meant to produce derived data, and typically the MC ones are very small, I would always keep both signal and background and eventually choose only one of the two later on in the task.